documentation/general/general_setup.md

```diff
@@ -1,263 +1,230 @@
-command.
+```{toctree}
+:maxdepth: 1
+:glob:
 
-```bash
-python3 --version
+/frontend/guides/quickstart.md
+/api/guides/quickstart.md
+/catalog/guides/quickstart.md
+/ingestion_server/guides/quickstart.md
 ```
 
-If you see `Python x.y.z`, you have Python installed. If you see an error, you
-need to install it using [pyenv](https://github.com/pyenv/pyenv) or by following
-the [official instructions](https://www.python.org/downloads/).
+## `ov` usage
 
-#### Pipenv
+Run `ov help` to read basic documentation for how to use and interact with `ov`.
 
-Pipenv helps us provision dependencies for our Python packages and automatically
-set up a virtualenv, provided you have the version of Python mentioned in the
-`Pipfile` installed and accessible locally.
+### Aliases
 
-You can install Pipenv by following the
-[official instructions](https://pipenv.pypa.io/en/latest/installation.html#installing-pipenv).
+`ov` has a number of built-in aliases, which are documented and discoverable by
+running `ov aliases`.
 
-### Node.js
+You may configure your own aliases. Run `ov aliases --help` for documentation.
+`ov init` includes an example alias in the generated `.ov_profile.json` to help
+you get started.
 
-```{note}
-This is only needed if you are working with the following:
+### Included tools
 
-- frontend
-- Node.js automations
-```
+The containerised `ov` environment includes a host of excellent general and
+specific tools to interact with Openverse's development environment. Some of
+them are listed here for visibility and awareness.
 
-We use Node.js in the frontend of our stack. So to work with that part of the
-codebase, you will need Node.js installed. To know if you have Node.js
-installed, run the following command.
+- [`just`](https://github.com/casey/just) is a command runner. Like `make` but
+  with loads of features for writing scripts `make` was never made for. Try
+  `ov just` to see all available recipes.
+- [`jq`](https://stedolan.github.io/jq/) is a tool for parsing and manipulating
+  JSON data. Openverse loves `jq`! Try `ov jq --help` for information.
+- [`pdm`](https://pdm-project.org/en/stable/) is a Python package manager. You
+  can use it in any Openverse directory to interact with `pyproject.toml` and
+  different virtual environments. Try `ov pdm --help` for info.
+- [`httpie`](https://httpie.io/docs/cli) is a command line HTTP client. Like
+  cURL but with a lot of conveniences and easier to remember. Try `http --help`
+  for info.
 
-```bash
-node --version
-```
+````{admonition} Bring your own tools
+:class: tip
 
-If you see `vx.y.z`, you have Node.js installed. If you see an error, you need
-to install it using [nvm](https://github.com/nvm-sh/nvm) or by following the
-[official instructions](https://nodejs.org/en/download/).
+Generally speaking, you should be able to rely on any tools you have on your host system.
+You can pipe output to and from `ov` and it should "just work". For example, the following
```

Comment:
Nit: I hate to use "just" as an adverb so close to references to the "just" command, but I can't think of an appropriate alternative here.

Reply:
I absolutely despise the name of the tool for this reason, I'm constantly second guessing when writing about it.